internal/rules/aws/ec2/no_excessive_port_access.go

```diff
@@ -15,7 +15,7 @@ var CheckNoExcessivePortAccess = rules.Register(
 		Provider:    providers.AWSProvider,
 		Service:     "ec2",
 		ShortCode:   "no-excessive-port-access",
-		Summary:     "An ingress Network ACL rule allows ALL ports.",
+		Summary:     "An egress Network ACL rule allows ALL ports.",
```

Comment:
This check is supposed to be about ingress. I think changing to egress would be a completely different rule. Am I missing something?

Reply:
If you look at impact, it is referring to egress. Egress is more important when talking about NACLs since thats where data leakage would come from, and in CSPM we have the same check that is looking specifically for egress. I can change it back to ingress but the logic in the plugin was not checking ingress/egress anyways. 

what do you think?

Reply:
I guess the impact is wrong as the severity is critical. I agree that egress is also important, but I don't think it is a critical issue, while ingress is critical. I'm not sure how the check was added originally. But it is a good idea to get aligned with our CSPM.
We can also hear @simar7's thoughts.

Reply:
here is the cloudsploit check for reference: https://github.com/aquasecurity/cloudsploit/blob/master/plugins/aws/ec2/networkAclOutboundTraffic.js

Reply:
Interesting. I thought ingress was more important than egress, but CloudSploit thinks the opposite as they have no check for nacl ingress. You can decide then. I'm ok with the current change.

Reply:
I think you both are right - so there are two things I found:

1. We have a rule for ingress: https://github.com/aquasecurity/defsec/blob/master/internal/rules/aws/ec2/no_public_ingress_acl.go
2. As for severity, Ingress to me certainly sounds CRITICAL as Teppei mentioned. But for egress, EC2 instances start wide open by default to my knowledge (no restrictions on egress traffic). Based on that, maybe we could lower the severity to HIGH or MEDIUM, but it is subjective as there isn't any guideline or spec to follow.

Reply:
Thanks, Simar. According to the rule Simar found, I guess this check is also for ingress originally.

>1. We have a rule for ingress: >https://github.com/aquasecurity/defsec/blob/master/internal/rules/aws/ec2/no_public_ingress_acl.go

But egress is also important as @giorod3 said. I agree we should have both. There are two options now.

1. Apply this check to both ingress and egress as it is now
    - In this case, we just add `rule.Action.EqualTo("allow") &&` in order to fix the original bug.
2. Split this check into two checks for having different severities
    - ingress: critical
    - egress: medium or high

@giorod3 should be the most familiar with CSPM. You can decide!